### PR TITLE
[WFLY-12829] Quickstart for MP OpenTracing 1.3

### PR DESCRIPTION
**Pull request by xstefank**
https://issues.redhat.com/browse/WFLY-12829

### CODE REVIEW

File: microprofile-opentracing/README.adoc [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+@GET
+@Path("/notTraced")
+@Produces(MediaType.TEXT_PLAIN)
+@Traced(false)
+public String notTraced() {
+    return "notTraced";
+}
+----
+
+And rebuild and deploy the application
```
[the remark this answers is not in the document]
Comment (xstefank) [reply microprofile-opentracing/README.adoc]: Yes, this is intentional. Automatic import resolution should be able to handle the import.

File: microprofile-opentracing/README.adoc [hunk 2]
```diff
@@ -0,0 +1,448 @@
...
+}
+----
+
+Note the explicit `@Traced` annotation on the `getHello()` method.
+
+And update the `TracedResource` to use our newly defined bean:
+
+[source,java]
+----
+@Inject
```
[the remark this answers is not in the document]
Comment (xstefank) [reply microprofile-opentracing/README.adoc]: Yes, this is also intentional. I would expect that the automatic import resolution should be able to handle this. If we receive reports that it is not, I will update it everywhere.

File: microprofile-opentracing/README.adoc [hunk 3]
```diff
@@ -0,0 +1,448 @@
...
+
+// Start the {productName} Standalone Server
+include::../shared-doc/start-the-standalone-server.adoc[leveloffset=+1]
+
+== Creating the Maven Project
+
+[source,options="nowrap"]
+----
+mvn archetype:generate \
+    -DgroupId=org.wildfly.quickstart.microprofile.config \
```
[the remark this answers is not in the document]
Comment (xstefank) [reply microprofile-opentracing/README.adoc]: thanks!

File: microprofile-opentracing/src/test/java/org/wildfly/quickstart/microprofile/opentracing/MicroProfileOpenTracingIT.java [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.core.Response;
+import java.net.URL;
+import java.util.List;
+
+/**
+ * Simple tests for MicroProfile OpenTracing quickstart. Arquillian deploys an JAR archive to the application server, which
+ * contains several REST endpoint and verifies that they are correctly invoked and the created spans are collected.
+ *
+ * @author <a href="mstefank@redhat.com>Martin Stefanko</a>
```
Comment (rhusar) [inline microprofile-opentracing/src/test/java/org/wildfly/quickstart/microprofile/opentracing/MicroProfileOpenTracingIT.java]: Missing "

Comment (xstefank) [reply microprofile-opentracing/src/test/java/org/wildfly/quickstart/microprofile/opentracing/MicroProfileOpenTracingIT.java]: thanks

File: microprofile-opentracing/pom.xml [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+      <scope>provided</scope>
+    </dependency>
+    <!-- Import the CDI API, we use provided scope as the API is included in WildFly -->
+    <dependency>
+      <groupId>jakarta.enterprise</groupId>
+      <artifactId>jakarta.enterprise.cdi-api</artifactId>
+      <version>2.0.2</version>
+      <scope>provided</scope>
+    </dependency>
+    <!-- Import the Jakarta REST API, we use provided scode as the API is included in WildFly -->
```
Comment (rhusar) [inline microprofile-opentracing/pom.xml]: s/scode/scope

**Pull request merged**